SPECS/cloud-hypervisor/cloud-hypervisor.spec

```diff
@@ -19,9 +19,15 @@ Source0:        https://github.com/cloud-hypervisor/cloud-hypervisor/archive/ref
 #   cd %{name}-%{version}
 #   cargo vendor > config.toml
 #   tar -czf %{name}-%{version}-cargo.tar.gz vendor/
+# rename the tarball to %{name}-%{version}-cargo.tar.gz when updating version
 Source1:        %{name}-%{version}-cargo.tar.gz
 Source2:        config.toml
 Patch0:         CVE-2023-45853.patch
+Patch1:         CVE-2023-50711-vmm-sys-util-fix1.patch 
```

Comment:
NIT: [Non blocking cosmetic suggestion]

We could club `Patch1 & Patch2` in a single file & similarly club `Patch3 to Patch5` again in a single file.

Reply:
I clubbed patches 1 and 2 as they are in the same package.